content/concept-amendments.md

```diff
@@ -191,6 +196,17 @@ A transaction remains in the queue until one of the following happens:
 * It becomes invalid (for example, the [`LastLedgerSequence`](reference-transaction-format.html#lastledgersequence) causes it to expire)
 * It gets dropped because there are too many transactions in the queue with a higher transaction cost.
 
+## fix1201
+
+| Amendment ID                                                     | Status    |
+|:-----------------------------------------------------------------|:----------|
+| B4D44CC3111ADD964E846FC57760C8B50FFCD5A82C86A72756F6B058DDDF96AD | Enabled   |
+
+Correctly implements a limit on [transfer fees](concept-transfer-fees.html) to a 100% fee, represented by a maximum `TransferRate` value of `2000000000`. (A 100% fee in this case means you must send 2 units of the issued currency for every 1 unit you want to deliver.) Without the amendment, the effective limit is a `TransferRate` value of 2<sup>32</sup>-1, for approximately a 329% fee.
+
+With this amendment enabled, an [AccountSet][] transaction that attempts to set `TransferRate` higher than `2000000000` fails with the result code `temBAD_TRANSFER_RATE`. Any existing `TransferRate` already set to a value higher than the limit is treated as equal to the limit instead.
```

Comment:
aww man, old grandfathered high fees still work?

I'm starting to want to put together a scavenger hunt for ledger objects in the production ledger that couldn't be created under the current rules...

Reply:
It would be really cool to make a list like that and let the data folks loose on it to see if any of those objects are actually hanging around in the MainNet.  But we'd have to be careful about removing support altogether since we still need to be able to replay old ledgers.